programs/pluto/kernel_xfrm.c

```diff
@@ -505,6 +548,9 @@ static bool sendrecv_xfrm_msg(struct nlmsghdr *hdr,
 		if (rsp.u.e.error != 0) {
 			llog_error(logger, -rsp.u.e.error,
 				   "netlink response for %s %s", description, story);
+			LLOG_JAMBUF(RC_LOG, logger, buf) {
+				print_ext_ack(&rsp.n, buf);
+			}
```

Comment:
I see print_ext_ack() can return without emitting anything which could lead to blank log and debug lines.  Can I suggest changing `print_ext_ack()` to:
```
  llog_ext_ack(lset_t rc_flags, struct logger *logger, const struct nlmsghdr *n)
```
with call:
```
llog_ext_ack(RC_LOG, logger, &rsp.n);
```
and ...